.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, in
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Summary
+<!--
+    Just give us a overview of your change, don’t go into technical details yet.
+-->
+
+:beginner: Describe this change in 1-2 sentences
+
+
+# Problem
+<!--
+    Anything that helps us understand why you are making this change goes here.
+    What problem are you trying to fix? What does this change address?
+-->
+
+:beginner: Reference related tickets with MBS-XXX at least.
+
+
+# Solution
+<!--
+    Talk about technical details, considerations, or other interesting points.
+-->
+
+:beginner: If you have a lot to say, be more detailed in this section.
+
+
+# Readiness
+<!--
+    The tasks you have to do to get your change ready for review. Use this if
+    you draft a pull request. Mark done tasks with an [x] as you progress. See
+    https://github.blog/2019-02-14-introducing-draft-pull-requests/
+-->
+
+* [x] Read the [Contributing Guidelines](https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md)
+* [ ] Delete irrelevant parts of this template
+* [ ] ...
```

Comment:
Is it worth having some standard items in here like "Write tests" or "Get the PR reviewed" ?

Reply:
* “Write tests” is a good suggestion but that should be detailed in contributing guidelines first.
* “Get the PR reviewed” doesn’t need to be done to be ready for review! :smile:
    I’m not sure newcomers are allowed to ask for review either.

Reply:
Finally added items for documentation:
- for author to edit the wiki: https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/410198fd412dd45e62862f79effbc293d55fc4c7#diff-195a635ad245ded9076330e31134bd80R51-R52
- for merger to transclude the doc: https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/410198fd412dd45e62862f79effbc293d55fc4c7#diff-195a635ad245ded9076330e31134bd80R62